packages/react-core/src/components/Wizard/examples/WizardValidateOnButtonPress.tsx

```diff
@@ -182,7 +174,6 @@ export const WizardValidateButtonPress: React.FunctionComponent = () => {
           return (
             <>
               <Button onClick={() => validateLastStep(onNext)}>Validate</Button>
-              <Button onClick={() => goToStepByName('First step')}>Go to Beginning</Button>
```

Comment:
Is there a reason this couldn't just be a "back" button? Maybe there is, but it feels like you're trapped on that step, even though you aren't and can still navigate using the nav.